.github/workflows/add-catalog.yml

```diff
@@ -26,122 +30,13 @@ jobs:
-              echo "$updatedPatternType doesn't exist."
-              echo "Creating directory... $updatedPatternType"
-              mkdir "./collections/_catalog/"$updatedPatternType""
-            fi
-
-            if [[ $patternInfo == "null" ]]; then
-              patternInfo="\"\""
-            fi
-
-            if [[ $patternCaveats == "null" ]]; then
-              patternCaveats="\"\""
-            fi
-
-            compatLength=$(jq -r ".[$idx].catalog_data.compatibility | length" temp.json)
-
-            for (( compatIdx=0; compatIdx<$compatLength; compatIdx++ ))
-            do
-              compatibility+="
-              "
-              compatibility+="    - $(jq -r ".[$idx].catalog_data.compatibility[$compatIdx]" temp.json)"
-            done
-            echo "$patternFile" > $MESHERY_CATALOG_FILES_DIR/$designId.yaml
-            services=$(yq '.services' $MESHERY_CATALOG_FILES_DIR/$designId.yaml 2>/dev/null)
-            if [[ $services == "null" || $services == "" ]]; then
-              patternImageURL="/assets/images/logos/service-mesh-pattern.svg"
-            fi
-            userId=$(jq -r .[$idx].user_id temp.json)
-            userInfo=$(curl -sS "$MESHERY_CLOUD_BASE_URL/api/identity/users/profile/$userId" 2>./err1.txt)
-
-            if [[ "$(<err1.txt)" != "" ]]; then
-            >&2 echo "Error fetching User details"
-            >&2 echo "$(<err1.txt)"
-            fi
-
-            echo "$userInfo" > info.json
-
-            userFullName="$(jq -r .first_name info.json) $(jq -r .last_name info.json)"
-            userAvatarURL="$(jq -r .avatar_url info.json)"
-
-          echo "---
-          layout: item
-          name: $patternName
-          userId: $userId
-          userName: $userFullName
-          userAvatarURL: $userAvatarURL
-          type: $patternType
-          compatibility: $compatibility
-          patternId: $designId
-          image: $patternImageURL
-          patternInfo: |
-            $patternInfo
-          patternCaveats: |
-            $patternCaveats
-          URL: 'https://raw.githubusercontent.com/meshery/meshery.io/master/$MESHERY_CATALOG_FILES_DIR/$designId.yaml'
-          downloadLink: $designId.yaml
-          ---" > ./collections/_catalog/"$(echo $patternType | tr '[:upper:]' '[:lower:]')"/$designId.md
-          curl -L \
-          -X POST \
-          -H "Accept: application/vnd.github+json" \
-          -H "Authorization: Bearer ${{ secrets.GH_ACCESS_TOKEN }}" \
-          -H "X-GitHub-Api-Version: 2022-11-28" \
-          "https://api.github.com/repos/meshery/meshery.io/actions/workflows/meshmap.yml/dispatches" \
-          -d '{"ref":"master","inputs":{"contentID":"'$designId'","assetLocation":"'$patternImageURL'"}}'
-          
-          done
-          rm temp.json
-          rm err.txt
-          rm info.json
-          rm err1.txt
+          cd assets/actions
+          patternId=$(go run package.go)
```

Comment:
What does it do? Is this required?

Reply:
@MUzairS15, ultimately, should this be an invocation of `mesheryctl design export`?

Reply:
It's just for this:
https://github.com/meshery/meshery.io/blob/1a3d0304e0e0130aef859860232ec63024040b77/.github/workflows/add-catalog.yml#L8C1-L12C72
by `echo "contentID=$patternId" >> $GITHUB_OUTPUT`

Reply:
Yes, not required now.